src/ts/view/ItemManageTab.js

```diff
@@ -0,0 +1,166 @@
+    this.itemInfoInputs = selectDoms('.item-info-input', this.itemInfoForm);
+    this.itemStatusTable = selectDom('.item-status-table', this.tabContent);
+
+    this.itemInfoForm.addEventListener('submit', this.#onSubmitItemInfoForm);
+    this.itemStatusTable.addEventListener('click', this.#onClickItemStatusTableButton);
+    this.itemStatusTable.addEventListener('keydown', this.#onKeyDownItemInfoRow);
+  }
+
+  #onClickItemManageTabButton = () => {
+    if (this.itemManageTabButton.classList.contains('selected')) {
+      return;
+    }
+    this.renderInitialTabState();
+  };
+
+  #onSubmitItemInfoForm = (e) => {
+    e.preventDefault();
+
+    const itemInfo = this.#convertToItemInfoObject(Array.from(this.itemInfoInputs));
+
+    try {
+      this.vendingMachine.validateItemInput(itemInfo);
+    } catch (error) {
+      return alert(error.message);
+    }
+
+    const newItem = this.vendingMachine.addItem(itemInfo);
+    this.#renderAddedItem(newItem);
+
+    this.itemInfoInputs.forEach((itemInfoInput) => (itemInfoInput.value = ''));
+    this.itemInfoInputs[0].focus();
+  };
+
+  #onClickItemStatusTableButton = ({ target }) => {
+    const targetItem = target.closest('tr');
+    if (!targetItem) {
+      return;
+    }
+
+    if (this.#isEditItemButton(target)) {
+      this.#handleEditButtonClickEvent(targetItem);
+      return;
+    }
+
+    if (
+      this.#isDeleteItemButton(target) &&
+      confirm(generateComfirmMessage(targetItem.dataset.itemName))
+    ) {
+      this.#handleDeleteButtonClickEvent(targetItem);
+      return;
+    }
+
+    if (this.#isConfirmItemButton(target)) {
+      this.#handleConfirmButtonClickEvent(targetItem);
+    }
+  };
+
+  #onKeyDownItemInfoRow = ({ key, target }) => {
+    const targetItem = target.closest('tr');
+
+    if (key === 'Enter' && !!targetItem) {
+      this.#handleConfirmButtonClickEvent(targetItem);
+    }
+  };
+
+  #handleEditButtonClickEvent(targetItem) {
+    const itemInfoInputCellList = selectDoms('.item-info-input-cell', targetItem);
+    const itemButtonCellList = selectDoms('.item-button-cell', targetItem);
+
+    itemInfoInputCellList[0].focus();
+    this.#toggleEditMode(itemInfoInputCellList, itemButtonCellList, false);
+    targetItem.addEventListener('keydown', ({ key }) =>
+      this.#onKeyDownItemInfoRow(key, targetItem)
+    );
+  }
+
+  #handleDeleteButtonClickEvent(targetItem) {
+    const { itemName } = targetItem.dataset;
+
+    this.vendingMachine.deleteItem(itemName);
+    targetItem.remove();
+  }
+
+  #handleConfirmButtonClickEvent(targetItem) {
+    const itemInfoInputCellList = selectDoms('.item-info-input-cell', targetItem);
+    const itemInfo = this.#convertToItemInfoObject(Array.from(itemInfoInputCellList));
+    const itemButtonCellList = selectDoms('.item-button-cell', targetItem);
+
+    try {
+      this.vendingMachine.validateItemInput(itemInfo, false);
+    } catch (error) {
+      return alert(error.message);
```

Comment:
alert는 반환값이 없어 아래 코드가 의미상으론 맞는 것 같습니다.
```suggestion
     alert(error.message);
     return ;
```